x-pack/plugins/fleet/common/openapi/components/schemas/package_info.yaml

```diff
@@ -131,6 +135,56 @@ properties:
             type: array
             items:
               type: string
+  owner: 
```

Comment:
I found a few more package spec properties that weren't added to the openapi spec docs